### ISSUE TITLE
About extractive labels

### ISSUE THREAD

**Issue by 1029694141** [potsawee/longsum0#9]
Hi，I've got extractive labels problem that are pressing on me.

I follow the steps：
1. "generate_ext_labels_arxiv_data" from raw arxiv dataset(arxiv_train/val.pk.bin)，ps: It has been set  start_id/end_id :train(from to 203037) and val(from 0 to 6436) separately.
2. and then combined with arxiv_train/val.pk.bin

but after combined the arxiv_train/val.pk.bin data size are 291/437 kb separately，the val is bigger than train 
（I use above data to train the hier model：model is not improved）
Can you answer this question or point out my mistake ？ thank u！ @potsawee

**Comment by potsawee** [comment]
1) On my disk, `arxiv_{train/val/test}.pk.bin` are 7.1GB, 219MB, 219MB, respectively. Are you sure you did the initial data pre-processing correctly?

2) 291/437 kB sounds like your data consists of ~10 examples rather than 200k,6k examples. You could do a simple check to see how many examples there are by e.g. `len(data)` or `len(data_xtras)` or whatever you call your data variable. You should see this show 203,037 for training set and 6,436 for validation set (arxiv dataset).

**Comment by 1029694141** [comment]
After I deleted this line, it became correct：
“data = data[:10] # just to show an example”
